samples/document-snippets/instance.js

```diff
@@ -74,44 +67,33 @@ const snippets = {
       location: 'us-central1-b',
       storage: 'hdd',
     };
-    instance
-      .createCluster(clusterId, options)
-      .then(result => {
-        const newCluster = result[0];
-        // const operations = result[1];
-        // const apiResponse = result[2];
-      })
-      .catch(err => {
-        // Handle the error.
-      });
+    const [newCluster, operation, apiResponse] = await instance.createCluster(
+      clusterId,
+      options
+    );
     // [END bigtable_create_cluster]
   },
 
-  createAppProfile: (instanceId, clusterId, appProfileId, callback) => {
+  createAppProfile: async (instanceId, clusterId, appProfileId, callback) => {
     // [START bigtable_create_app_profile]
     const Bigtable = require('@google-cloud/bigtable');
     const bigtable = new Bigtable();
     const instance = bigtable.instance(instanceId);
-
+    const util = require('util');
     const cluster = instance.cluster(clusterId);
 
     const options = {
       routing: cluster,
       allowTransactionalWrites: true,
       ignoreWarnings: true,
     };
-
-    instance.createAppProfile(appProfileId, options, (err, appProfile) => {
-      if (err) {
-        // Handle the error.
-        return callback(err);
-      }
-      return callback(appProfile);
-    });
+    const instanceAddProfile = util.promisify(instance.createAppProfile);
```

Comment:
[`instance.createAppProfile()`](https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L156) should already be promisified in the `instance.js` module

https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L766-L768